.github/workflows/python-sdk-tutorial-auto-ml-classification-bank-marketing-all-features.yml

```diff
@@ -13,7 +13,7 @@ on:
       - .github/workflows/python-sdk-tutorial-auto-ml-classification-bank-marketing-all-features.yml
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Instead of hard coding the specific version of ubuntu in multiple files, we can configure a variable or environment to use for fetching the runner.

runs-on: ${{ vars.RUNNER }}

Benefit: You don't have to change multiple file next time when you are updating the agent pool.

Reply:
Just created a variable: V1_UBUNTU_RUNNER with a value as ubuntu-20.04

Reply:
Updated